src/qt/guiutil.cpp

```diff
@@ -414,7 +415,7 @@ void bringToFront(QWidget* w)
 
 void handleCloseWindowShortcut(QWidget* w)
 {
-    QObject::connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_W), w), &QShortcut::activated, w, &QWidget::close);
+    QObject::connect(new QShortcut(QKeySequence(QShortcut::tr("Ctrl+W")), w), &QShortcut::activated, w, &QWidget::close);
```

Comment:
0fea3846d0d6ec69dadd8bf043afd3c25971d07f

`QObject::tr()`

Reply:
The initial idea was to put this translation into the context named "QShortcut", rather "QObject". But context name does not go from `src/qt/locale/bitcoin_en.ts` into the Transifex environment. So [accepting](https://github.com/bitcoin/bitcoin/pull/24624#issuecomment-1074850209) your suggestion.